website/client/assets/scss/icon.scss

```diff
@@ -1,8 +1,6 @@
 .svg-icon {
   display: block;
   stroke-width: 0;
-  stroke: currentColor;
```

Comment:
I'm sure this is working but you first have to remove any fill="xxx" from the svg files before you can use it, otherwise it cannot be overriden